src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,27 +1,33 @@
 <script setup>
-import { ref } from 'vue'
+import { computed, ref } from 'vue'
+import { MaInput } from "@mobileaction/action-kit"
+import { MaBadge } from "@mobileaction/action-kit"
 
+const checkedNumbers = ref([])
+const excludeWords = ref('is, a, an, the') // default words to exclude
 const sentence = ref('')
-const MAX_N_GRAM_COUNT = 3
 const nGrams = ref([])
+const words = computed( () => {
```

Comment:
Splitting this computed property into two based on their purposes will help us write more readable code.

```
const excludedWordsArray = computed(() => {
  return excludeWords.value.split(/\s*,\s*/).map(word => word.toLowerCase());
});

const filteredWords = computed(() => {
  return inputText.value.split(/\s+/).filter(word => !excludedWordsArray.value.includes(word.toLowerCase()));
});
```

Reply:
also, the naming 'words' is not clear, so making such a change in naming will be more descriptive.